test/unit/client_interface_sections/search_test.rb

```diff
@@ -1,13 +1,21 @@
 require_relative '../../test_helper'
 
 class ClientInterfaceSearchTest < Test::Unit::TestCase
-  def client
-    @client ||= FHIR::Client.new("search-test")
+  def setup
+    @client = FHIR::Client.new("http://search-test")
+  end
+  def empty_search_response
+    {
+      headers: {
+        'Accept': 'application/fhir+json'
+      },
```

Comment:
What's the reason for the `Accept` header on the response? It doesn't look like it's used anywhere and typically a request header.